action-token-authenticator/src/test/java/org/keycloak/quickstart/ArquillianActionTokenWithAuthenticatorTest.java

```diff
@@ -71,18 +73,26 @@
 import static org.keycloak.test.TestsHelper.keycloakBaseUrl;
 
 @RunWith(Arquillian.class)
+@Ignore("https://issues.redhat.com/browse/KEYCLOAK-13055")
```

Comment:
Do NOT add @Ignore to tests

Reply:
Could you please explain why? I think this approach has a lot of advantages:
- Once we make the testsuite green, we will be able to quickly spot new failures. 
- Each test that requires revisiting has a corresponding JIRA.
- It's very easy to spot, why a specific test has been ignored.

If you don't like this approach, please suggest a better one.

Reply:
Well, from my PoV it all depends on whether we want to make this PR even bigger. :) And I suppose we don't. Even though I really hate ignoring tests, I must agree with @slaskawi here. The point of this PR is to make Travis green again so we can test at least something. And this way we're splitting the remaining broken tests into separate task so we don't need to review it as a part of gigantic PR as this one.